src/components/__tests__/FormFieldOptionGroup.cy.js

```diff
@@ -0,0 +1,85 @@
+import { colors } from 'quasar';
+import FormFieldTestWrapper from 'components/global/FormFieldTestWrapper.vue';
+import { i18n } from '../../boot/i18n';
+import { rideToWorkByBikeConfig } from '../../boot/global_vars';
+import { hexToRgb } from 'app/test/cypress/utils';
```

Comment:
We can replace this custom `hexToRgb` function with the Quasar `hexToRgb` function https://quasar.dev/quasar-utils/color-utils#introduction throughout the code if possible. You can do this in the next new PR.

Reply:
Updated. However, the function unfortunately does not return string, but an object.
So additional rewrite into rgb(xxx) value is needed to work in the test.
`const colorPrimary = 'rgb(${rgbPrimary.r}, ${rgbPrimary.g}, ${rgbPrimary.b})';`